### PR TITLE
feat(explorer): add dependabot

### LINKED ISSUE #685: build(explorer): add dependabot for security updates

### PR DESCRIPTION
**Pull request by glpecile**
Dependabot is a common practice to keep dependencies up-to-date.

This PR adds a dependabot, which checks for dependency updates, and makes a new PR updating them, if needed.

This PR limits dependabot PR creations to up to 1 per week, and up to 5 open PRs at the same time.

### CODE REVIEW

**Review by uri-99 — CHANGES_REQUESTED** [review]
This should be tested on this branch before pushed to main.
This can be done by changing the target-branch, making the interval higher, injecting an old version, and then seeing if Dependabot works ok.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: "mix"
+    directory: "/explorer"
+    open-pull-requests-limit: 5
+    schedule:
+     interval: "weekly"
+    target-branch: "main"
```
Comment (glpecile) [inline .github/dependabot.yml]:
```suggestion
    target-branch: "staging"
```

**Comment by glpecile** [comment]
@uri-99 the image below basically shows that the config is correct and will start publishing updates once merged. 

image

**Comment by Oppen** [comment]
Converted to draft due to not being a priority for mainnet

**Comment by JuArce** [comment]
Stale

**Pull request closed without merging**